lib/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(JPEGXL_LIBRARY_VERSION
 # It is important to update this value when making incompatible API/ABI changes
 # so that programs that depend on libjxl can update their dependencies. Semantic
 # versioning allows 0.y.z to have incompatible changes in minor versions.
-set(JPEGXL_SO_MINOR_VERSION 6)
+set(JPEGXL_SO_MINOR_VERSION 7)
```

Comment:
@jonsneyers I've inspected the change at the library level. All I can find is _new_ symbols being added to the library. In this case there is no need to bump the SONAME of the library. Could you confirm this is the case ?
Here is the documentation, I've used:
* https://docs.oracle.com/cd/E19253-01/817-1984/appendixb-45356/index.html#6mhm7plcq
Here is what I see on my side (git/master as of this morning):
```
 % readelf -d ./obj-x86_64-linux-gnu/libjxl.so | grep soname
 0x000000000000000e (SONAME)             Library soname: [libjxl.so.0.7]
```

Reply:
The non-opaque JxlFrameHeader struct changes so I think that requires a SONAME bump. Please correct me if I'm wrong though.

Reply:
Oh right, totally missed that change! Thanks for the quick answer.